### PR TITLE
VACMS-13218 Enable module documentation.

### LINKED ISSUE #13218: Enable ability to document modules
## Description
The content model documentation module now allows us to document modules, BUT it needs a config addition to enable it for us.

## Acceptance Criteria
- [x] Modules can be one of the things that can be documented.
- [x] Documentation on Documentation has been updated https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/READMES/content-models.md

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

closes #13218

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As an admin
1. Go to https://pr13219-nioxw47w7huwhxq1incy2daszuyj30fl.ci.cms.va.gov/admin/structure/cm_document/add
2. IN the select list "Documentation for:"  scroll down until you get to the module section of the list.  (alphabetical, in the "m")
   - [x] Validate that modules show up to be selected.

image

3. Then choose a module and create a tests document
   - [ ] Validate that it saves.

image

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/node_link_report.settings.yml [hunk 1]
```diff
@@ -79,9 +79,11 @@ domains_to_skip: |-
   www.teamusa.org
   www.usajobs.gov
   www.veteranscornerok.org
+  www.vietnamwar50th.com
```
Comment (swirtSJW) [inline config/sync/node_link_report.settings.yml:82]: These are unrelated changes that ride along because this setting for node link report is config ignored, so it floats a bit.

**Review by timcosgrove — APPROVED** [review]

**Comment by swirtSJW** [comment]
image
